getting-started/4_start_ledger.md

```diff
@@ -0,0 +1,27 @@
+We can now run the app in two steps. The first is to run the DAML Ledger. 
+
+Because we are running in a special environment, we need to configure the API to listen to external connections. To do so, we add the following two lines to the `daml.yaml`{{open}} project definition:
+
+<pre class="file" data-filename="daml.yaml" data-target="replace">json-api-options:
```

Comment:
Clicking on this doesn’t seem to work for me.

Reply:
Manual copy and paste also doesn’t seem to work. It looks like they hijack that somehow in a way that breaks it in my setup (firefox and weird keyboard layouts).

Reply:
Yeah, I have an open support ticket for this.